### DISCUSSION TITLE
Can't Upload Replit Files

### DISCUSSION THREAD

**Discussion by VolcanicGameStudios** [community/community#78709] [New to GitHub] [answered]
Basically I downloaded my entire Replit project because I wanted to switch to github. I have been trying to upload the files but it won't let me do that. What should I do?

**Comment by DylanDevelops** [comment]
Try creating the repository on GitHub and cloning it to your computer. Then add all the files to that local version of your repository. After it's all set up correctly, push it back up to GitHub. 

If you aren't as familiar with the command line process, you can do this intuitively with GitHub Desktop, which you can download at [desktop.github.com](https://desktop.github.com).

**Comment by imrashidov** [comment, marked as answer]
You should be able to push code from Replit to GitHub. Please see below steps that you can try to connect the accounts.

1. Open the Replit project that you want to push to GitHub.
2. Click on the "Version Control" tab on the left-hand side of the Replit window.
3. Click on the "Connect to GitHub" button and follow the prompts to authenticate with your GitHub account.
4. Once you've connected your Replit project to GitHub, you should see a list of your GitHub repositories in the "Version Control" tab.

**Comment by anujns** [comment]
I am unable to connect to Github from Replit. Replit repeatedly requests access to connect to Github repos, which I approve. But nothing changes on the Github side. Replit gives an error:

Git Error
UNAUTHENTICATED
Failed to authenticate with the remote. Please ensure that Replit has access to this repository on GitHub.

**Reply by StayMeaty** [reply]
Got the same Issue rn!

**Reply by deeduckme** [reply]
same issue

**Reply by curiorevelio** [reply]
Same issue, and replit support is also not able to solve this.

i doubting they want to solve this at all, trying to avoid user transitioning to other no code apps
